tests/queries/0_stateless/00623_truncate_all_tables.sql

```diff
@@ -1,5 +1,3 @@
--- Tags: no-parallel
-
 DROP DATABASE IF EXISTS truncate_test;
```

Comment:
I guess this test was setup as no-parallel because it creates a database, so if it's run in parallel with itself it will fail.

We could change this test to a `.sh` and use `$CURRENT_DATABASE` in the `TRUNCATE ALL TABLES` call, removing the need of creating and dropping a database manually, which is good for performance and consistency (and we can remove all the `truncate_test.`)

Reply:
It's not necessary to rewrite in bash: https://github.com/ClickHouse/ClickHouse/blob/3fc65a68c7cd6aabea5250784ee068014ffaa801/tests/queries/0_stateless/00080_show_tables_and_system_tables.sql#L2